ambry-api/src/main/java/com.github.ambry/clustermap/ClusterManagerWriteStatusDelegate.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.clustermap;
+
+/**
+ * Delegate class giving a layer of abstraction between Store and ClusterParticipant,
```

Comment:
minor: `Store` does not need to be mentioned at all maybe?

Reply:
How about "Delegate class giving a layer of abstraction between the client and ClusterParticipant" ?